src/shared/components/pipeline-run-logs/logs/Logs.tsx

```diff
@@ -46,16 +47,22 @@ const Logs: React.FC<LogsProps> = ({
 
   const appendMessage = React.useRef<(blockContent) => void>();
 
+  const safeScroll = throttle(() =>
+    setTimeout(() => {
```

Comment:
why is this done in a timeout; is it even necessary? If so why not `requestAnimtationFrame` ?

btw it looks like ODC throttles updates every 300ms.

Reply:
I wrapped in setTimeout to make the call async and to make the scroll experience smoother, now switched to `requestAnimationFrame` and added 300 ms throttling as well.